pt-BR/mail/chrome/messenger/preferences/sendoptions.dtd

```diff
@@ -9,18 +9,18 @@
 <!ENTITY sendHTML.accesskey           "E">
 <!ENTITY sendBoth.label               "Enviar junto uma versão sem formatação">
 <!ENTITY sendBoth.accesskey           "n">
-<!ENTITY receiveHTML.label            "If you know a particular recipient can receive HTML mail, uncheck the 'Send email as plain text (no HTML)' checkbox on the recipient's Address Book card.">
+<!ENTITY receiveHTML.label            "Se você souber que um determinado destinatário pode receber e-mails em HTML, desmarque a caixa 'Enviar e-mail como texto sem formatação (sem HTML)' no cartão do destinatário disponível no catálogo de endereços.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY receiveHTML.label            "Se você souber que um determinado destinatário pode receber mensagens em HTML, desmarque a caixa 'Enviar e-mail como texto sem formatação (sem HTML)' no cartão do destinatário, disponível no catálogo de endereços.">